docs/Setup/UpgradeFrom3to3.x.x.md

```diff
@@ -0,0 +1,48 @@
+---
+currentMenu: upgrade3to3
+---
+# Upgrading Zikula 3.x.x to Zikula 3.x.x
+This document is the upgrade process for **Zikula Core 3.x.x to 3.x.x**. If you are upgrading from a version **less** than
+3.0.0 please use the proper [upgrade document](Upgrade.md). 
+
+1. [Test Environment](#test-environment)
+2. [Before upgrading](#download)
+3. [Upgrading](#upgrading)
+
+## Backup your current files and database!
+***Prior to any upgrade ensure that you have created a reliable backup of all files and the database.***
+
+## Test Environment
+The Zikula team strongly recommends having a duplicate testing environment of the live site in which all
+changes including upgrades are tested on before application to the live site.
+
+## Download
+Download the current release from [GitHub releases](https://github.com/zikula/core/releases/)
+All the dependencies and requirements are included in this package.
+
+## Upgrading
+- Before uploading the new files, delete (or move) **all files** in your web root (typically `public_html` or `httpdocs`).
+- Upload the new package and unpack the archive.
+  - Please read the [INSTALL docs](INSTALL.md#upload) for detailed information on proper uploading.
+  - Ensure `/var/cache` and `/var/log` directories are writable both by the web server and the command line user.
+  - If you have `mod_suexec` installed for Apache the CLI will run into permission problems. (If you are not sure 
+    check your phpinfo.) `mod_suexec` often is used in shared hosting environments. In this case, the CLI upgrader is not 
+    recommended, please use the Web Upgrade.
+- Copy old files to parallel locations:
+    - Copy your previous installation's `/.env.local` to `/.env.local` in your new installation.
```

Comment:
Maybe add: `Do the same for any `/.env_{env}.local` file which may exist in addition.` ?

Reply:
like what? the test envs are not needed. there are no others.